SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -112,6 +112,8 @@ namespace SpatialConstants
 	const Worker_ComponentId SERVER_RPC_ENDPOINT_COMPONENT_ID				= 9989;
 	const Worker_ComponentId NETMULTICAST_RPCS_COMPONENT_ID					= 9987;
 	const Worker_ComponentId NOT_STREAMED_COMPONENT_ID						= 9986;
+	const Worker_ComponentId SERVER_PING_COMPONENT_ID						= 9985;
```

Comment:
Assuming the strange indentation is to group things, then presumably the pings should not be grouped with the streamed component.

Reply:
Misreading this as github resolves tabs differently from editors.